### PR TITLE
Dapr presentation slide deck

### PR DESCRIPTION
**Pull request by nyemade-uversky**

### CODE REVIEW

File: running-dapr/releases/release-process.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
 # Release cycle and cadence
```
Comment (msfussell) [inline running-dapr/releases/release-process.md:1]: If you are editing this file, I would put an intro paragraph here to the release cycle purpose. This is currently missing.

File: running-dapr/community-calls/existing-hosts.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+## List of existing community meeting hosts
+When running a dapr community call you'll often need the help of existing community call hosts (i.e., start zoom call, agenda coordination help, etc.). This file lists all existing community call hosts that can help when running a dapr community call. 
+
+### Most recent hosts
+| **Host name**  |  **Github handle** | **Discord handle**  |  
+|:---|:---|:---|
+| Nick Greenfield  | greenie-msft  | nickgreenfield   |
```
[the remark this answers is not in the document]
Comment (nyemade-uversky) [reply running-dapr/community-calls/existing-hosts.md]: Removed names

File: running-dapr/community-calls/Hosting-community-meetings.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+- [ ] **Confirm zoom call details with existing hosts.** Confirm that an existing host is avilable to start and own the official Dapr Zoom account during the community meeting. 
+
+### Hosting community call (day-of)
+On the day of the community meeting zoom call, join using the link found on the main community page, [community meeting zoom links](https://github.com/dapr/community#community-meetings). Make sure all speakers are able to join, and that the slide deck is ready to go.
+
+#### Day-of hosting checklist
+- [ ] **Join community meeting zoom link.** Join the community call 15 minutes early to confirm final details (i.e., 8:45 PT). Existing host or community maintainers should have already started the official Dapr zoom account for you to join the meeting. If Zoom call hasn't been started, ping an existing host. Existing host list and information can be found here, [existng community call hosts](existing-hosts.md). 
+- [ ] **Confirm that slide deck is ready.** Check the slide deck for the community call to make sure it matches the agenda discussed on the _Community Call Agenda_ issue. Also check that each presenter followed the template.
```
[the remark this answers is not in the document]
Comment (nyemade-uversky) [reply running-dapr/community-calls/Hosting-community-meetings.md:37]: done

File: running-dapr/presentations/dapr-slide-deck.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+## Running presentations about Dapr
+The Dapr project provides a slide deck presentation slide that can be used for discissions on Dapr. The full presentation slide deck can be found here, [Dapr Presentation Slide Deck](https://docs.google.com/presentation/d/1U8PY2j0ew10n1yXKxF9lTLTRJ6OLSscn/edit?usp=sharing&ouid=115924791314872397226&rtpof=true&sd=true).
+
+#### This presentation serves three purposes:
```
[the remark this answers is not in the document]
Comment (nyemade-uversky) [reply running-dapr/presentations/dapr-slide-deck.md:4]: done

**Comment by nyemade-uversky** [comment]
@msfussell  all changes made. Let me know if we need anything else to get this merged - trying to clean up PRs in this repo

**Comment by msfussell** [comment]
I do not think that we should move the PPT deck to the community repo. What is the benefit of moving this from the doc repo?

**Review by nyemade-uversky — COMMENTED** [review]
I thought it made more sense in community since it's a slide deck intended for the dapr community to utilize in their presentations. That said, I'm fine just leaving it in the docs repo - I already moved it back to the docs repo.

**Pull request closed without merging**